### ISSUE TITLE
DateTime support for Android

### ISSUE THREAD

**Issue by gelodgreat** [react-native-datetimepicker/datetimepicker#945]
# Feature request

## Why it is needed
Is there a way to combine date and time on android like the same on iOS? 

## Possible implementation

We're using this for events that would be better if we can just pass them as one value

**Comment by sidorchukandrew** [comment]
No, Android does not provide this functionality with it's core date and time picker components. See [these Android docs](https://developer.android.com/develop/ui/views/components/pickers) to see how the pickers work. The intention of this library is to expose the native functionality to React Native and so we are limited to what Android provides.
